app/src/main/java/ch/epfl/skysync/components/ChatText.kt

```diff
@@ -110,35 +105,32 @@ fun ChatBubble(message: ChatMessage, index: String) {
         }
       }
   Row(
```

Comment:
why is there a Row composable if it contains only one composable? For the alignment?

Reply:
yes